packages/components/fields/time-field/src/time-field.tsx

```diff
@@ -0,0 +1,262 @@
+import {
+  Component,
+  isValidElement,
+  type KeyboardEvent,
+  type MouseEvent,
+  type ReactElement,
+  type ReactNode,
+} from 'react';
+import {
+  filterDataAttributes,
+  createSequentialId,
+  getFieldId,
+  warning,
+} from '@commercetools-uikit/utils';
+import Constraints from '@commercetools-uikit/constraints';
+import Stack from '@commercetools-uikit/spacings-stack';
+import FieldLabel from '@commercetools-uikit/field-label';
+import TimeInput from '@commercetools-uikit/time-input';
+import FieldErrors from '@commercetools-uikit/field-errors';
+
+type TEvent = {
+  target: {
+    id?: string;
+    name?: string;
+    value?: unknown;
+  };
+  persist?: () => void;
+};
+
+type TFieldErrors = Record<string, boolean>;
+
+type TTimeFieldProps = {
+  // TimeField
+  /**
+   * Used as HTML id property. An id is auto-generated when it is not specified.
+   */
+  id?: string;
+
+  /**
+   * Horizontal size limit of the input fields.
+   */
+  horizontalConstraint?:
+    | 3
+    | 4
+    | 5
+    | 6
+    | 7
+    | 8
+    | 9
+    | 10
+    | 11
+    | 12
+    | 13
+    | 14
+    | 15
+    | 16
+    | 'scale'
+    | 'auto';
+
+  /**
+   * A map of errors. Error messages for known errors are rendered automatically.
+   * <br/>
+   * Unknown errors will be forwarded to renderError.
+   */
+  errors?: TFieldErrors;
+
+  /**
+   * Called with custom errors, as renderError(key, error). This function can return a message which will be wrapped in an ErrorMessage.
+   * <br />
+   * It can also return null to show no error.
+   */
+  renderError?: (key: string, error?: boolean) => ReactNode;
+
+  /**
+   * Indicates if the value is required. Shows an the "required asterisk" if so.
+   */
+  isRequired?: boolean;
+
+  /**
+   * Indicates whether the field was touched. Errors will only be shown when the field was touched.
+   */
+  touched?: boolean;
+
+  // TimeInput
+  /**
+   * Used as HTML name of the input component.
+   */
+  name?: string;
+
+  /**
+   * Used as HTML autocomplete of the input component.
+   */
+  autoComplete?: string;
+
+  /**
+   * Value of the input
+   */
+  value: string;
+
+  /**
+   * Called with an event holding the new value.
+   * <br/>
+   * Required when input is not read only. Parent should pass it back as `value`-
+   */
+  onChange: (event: TEvent) => void;
+
+  /**
+   * Called when input is blurred
+   */
+  onBlur?: (event: TEvent) => void;
+
+  /**
+   * Called when input is focused
+   * <br/>
+   * Signature: `(event) => void`
```

Comment:
Nit: if I understand correctly we have that comment autogenerated now

Reply:
Fixed in 7df74c0512fbc91a527c94a3daf6f1563f2cb405

Thanks for letting me know about this one 👍